src/modules/Bootloader.rb

```diff
@@ -150,7 +152,21 @@ def Read
       Progress.NextStage
       return false if testAbort
 
-      ::Bootloader::BootloaderFactory.current.read
+      begin
+        ::Bootloader::BootloaderFactory.current.read
+      rescue ::Bootloader::UnsupportedBootloader => e
+        ret = Yast::Report.AnyQuestion(_("Unsupported Bootloader"),
+          _("System uses unsupported bootloader '%s'. Propose to use supported bootloader?") %
```

Comment:
well, we do not know it in advance, that I would have to computate and still user can change it. It is mainly to open bootloader module with something reasonable and do not crash.